cogs/polling.py

```diff
@@ -0,0 +1,124 @@
+import discord
+import asyncio
+from discord.ext import commands
+import random
+from discord.ext.commands.cooldowns import BucketType
+
+
+class Poll(commands.Cog):
+    def __init__(self, bot):
+        self.bot = bot
+
+        self.emojiLetters = [
```

Comment:
Do we need the full alphabet? I think we should limit it to A-F. We need to consider message length as well.